packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -1,553 +1,72 @@
-import {
-    formatNetworkAmount,
-    getPendingAccount,
-    getAreSatoshisUsed,
-    amountToSatoshi,
-    formatAmount,
-    getAccountDecimals,
-    hasNetworkFeatures,
-    isCardanoTx,
-} from '@suite-common/wallet-utils';
 import {
     FormState,
-    ComposeActionContext,
     PrecomposedTransactionFinal,
-    PrecomposedTransactionFinalCardano,
+    FormSignedTx,
+    AccountKey,
+    TxFinalCardano,
 } from '@suite-common/wallet-types';
-import { cloneObject, getSynchronize } from '@trezor/utils';
-
-import * as modalActions from 'src/actions/suite/modalActions';
-import * as metadataLabelingActions from 'src/actions/suite/metadataLabelingActions';
-import { SEND } from 'src/actions/wallet/constants';
-import { Dispatch, GetState } from 'src/types/suite';
-import { Account } from 'src/types/wallet';
-import { MetadataAddPayload } from 'src/types/suite/metadata';
-
-import * as sendFormBitcoinActions from './send/sendFormBitcoinActions';
-import * as sendFormEthereumActions from './send/sendFormEthereumActions';
-import * as sendFormRippleActions from './send/sendFormRippleActions';
-import * as sendFormSolanaActions from './send/sendFormSolanaActions';
-import * as sendFormCardanoActions from './send/sendFormCardanoActions';
-import { findLabelsToBeMovedOrDeleted, moveLabelsForRbfAction } from './moveLabelsForRbfActions';
-
-export type SendFormAction =
-    | {
-          type: typeof SEND.STORE_DRAFT;
-          key: string;
-          formState: FormState;
-      }
-    | {
-          type: typeof SEND.REMOVE_DRAFT;
-          key: string;
-      }
-    | {
-          type: typeof SEND.REQUEST_SIGN_TRANSACTION;
-          payload?: {
-              formValues: FormState;
-              transactionInfo: PrecomposedTransactionFinal;
-          };
-      }
-    | {
-          type: typeof SEND.REQUEST_PUSH_TRANSACTION;
-          payload?: {
-              tx: string;
-              coin: Account['symbol'];
-          };
-      }
-    | {
-          type: typeof SEND.SEND_RAW;
-          payload?: boolean;
-      }
-    | {
-          type: typeof SEND.DISPOSE;
-      };
-
-export const saveDraft = (formState: FormState) => (dispatch: Dispatch, getState: GetState) => {
-    const { selectedAccount } = getState().wallet;
-    if (selectedAccount.status !== 'loaded') return null;
-
-    dispatch({
-        type: SEND.STORE_DRAFT,
-        key: selectedAccount.account.key,
+import { createAction } from '@reduxjs/toolkit';
+import { MODULE_PREFIX } from './send/constants';
+
+const storeDraft = createAction(
+    `${MODULE_PREFIX}/store-draft`,
+    ({ accountKey, formState }: { accountKey: AccountKey; formState: FormState }) => ({
+        payload: {
+            accountKey,
+            formState,
+        },
+    }),
+);
+
+const removeDraft = createAction(
+    `${MODULE_PREFIX}/remove-draft`,
+    ({ accountKey }: { accountKey: AccountKey }) => ({
+        payload: {
+            accountKey,
+        },
+    }),
+);
```

Comment:
You should be able to return the payload just like this. 

```suggestion
const removeDraft = createAction(
    `${MODULE_PREFIX}/remove-draft`,
    ({ accountKey }: { accountKey: AccountKey }) => ({
            accountKey,
    }),
);
```

Reply:
Does not seem like that :/ 
![Screenshot 2024-02-22 at 10 48 25](https://github.com/trezor/trezor-suite/assets/26143964/c207dd4d-69e8-4c2d-b62b-a35ad0895206)

Reply:
```
const storeDraft = createAction(
    `${MODULE_PREFIX}/store-draft`,
    (payload: { accountKey: AccountKey; formState: FormState }) => ({ payload }),
);
```

Reply:
fixed by 177cf1b5cc